homepage/locales/es.ts

```diff
@@ -2,92 +2,93 @@ export default {
-      // extensibleMarkdown: 'Extensible Markdown',
-      // extensibleMarkdownDescription:
-      //   'You can introduce custom markdown syntax and configure how to render it.',
+      title: 'Características',
+      cloudStorage: 'Almacenamiento en la nube',
+      cloudStorageDescription:
+        'Las notas en almacenamiento en la nube serán guardadas de forma segura y accesibles desde otros dispositivos.',
+      multiplePlatforms: 'Multiples Platformas',
+      multiplePlatformsDescription:
+        'La aplicación Boost Note está disponible en navegadores, aplicación de escritorio y aplicación móvil.',
+      syntaxHighlight: 'Resaltado Sintáxis',
+      syntaxHighlightDescription:
+        'Boost Note puede resaltar más de 100 lenguajes de programación.',
+      mathEquations: 'Ecuaciones Matemáticas',
+      mathEquationsDescription:
+        'Boost Note soporta bloques matemáticos. En los bloques, puedes escribir con sintaxis de LaTeX.',
+      customizableTheme: 'Tema Personalizable',
+      customizableThemeDescription:
+        'Puedes personalizar el estilo de la interfaz tu aplicación, su editor y el contenido del Markdown renderizado.',
+      fileSystemBasedStorageDescription:
+        'Puedes tener control completo de todos tus datos. Comparte tus notas con tu servicio en la nube favorito',
+      extensibleMarkdown: 'Markdown Extensible',
+      extensibleMarkdownDescription:
+        'Puedes introducir sintaxis personalizada de Markdown y configurar cómo renderizarla.',
     },
     pricing: {
-      // title: 'Pricing Plans',
+      title: 'Planes de pago',
       feature: 'Característica',
       basic: 'Básico',
       premium: 'Premium',
-      sync: 'Sincronizar múltiples dispositivos',
+      sync: 'Sincronizando múltiples dispositivos',
       localStorageSize: 'Almacenamiento local',
       unlimited: 'Ilimitado',
       cloudStorageSize: 'Almacenamiento en la nube',
-      // price: 'Price',
-      // free: 'Free',
+      price: 'Precio',
+      free: 'Gratis',
       month: 'mes',
       furtherPlan: 'Después, $5.00 por cada 5GB extra.',
     },
     download: {
-      // legacyApp: 'Legacy App',
-      // legacyAppDescription:
-      //   "We are going to keep maintaining the old app until the current Boost Note.next support most of features of the old app like file system based storage and markdown extensions. So please don't force yourself too much to migrate to the new app.",
-      // legacyDownloadLinks: 'Legacy Download Links',
-      // legacyRepository: 'Legacy Repository',
+      legacyApp: 'Legacy App',
+      legacyAppDescription:
+        'Vamos a seguir manteniendo la aplicación vieja hasta que la actual Boost Note.next soporte la mayoría de las características de la aplicación vieja, cómo el sistema de almacenamiento basado en sistema de archivos y extensiones para Markdown. Así que por favor no se fuerce demasiado usted mismo a migrar a la nueva aplicación.',
```

Comment:
nit: What do think about changing "vieja" by "anterior"?

"Continuaremos manteniendo la aplicación anterior hasta que la actual Boost Note.next soporte la mayoría de las características de la aplicación anterior, cómo el sistema de almacenamiento basado en sistema de archivos y extensiones para Markdown. Así que por favor no se fuerce demasiado para migrar a la nueva aplicación.'

Reply:
It seems good to me.